blocks/product-details/product-details.js

```diff
@@ -150,6 +163,13 @@ export default async function decorate(block) {
           if (valid) {
             const { addProductsToCart } = await import('@dropins/storefront-cart/api.js');
             await addProductsToCart([{ ...values }]);
+
+            if (updateCartInfo && product) {
+              const cartUrl = new URL(rootLink('/cart'), window.location.origin);
+              cartUrl.searchParams.append('itemUid', updateCartInfo.itemUid);
+
+              window.location.href = cartUrl.toString();
+            }
```

Comment:
The redirect to cart page and the redirected URL have been handled in PR#419. So we could remove this and use the outcome from PR#419. 
 ```suggestion
```